projects/packages/tracking/src/class-tracking.php

```diff
@@ -99,6 +112,14 @@ public function enqueue_tracks_scripts() {
 			true
 		);
 
+		wp_enqueue_script(
```

Comment:
jp-tracks-functions should not be enqueued at this point. Doing so will add it to render in the footer regardless of if it is needed.